dev-docs/bidders/criteo.md

```diff
@@ -10,17 +10,17 @@ hide: true
 
 biddercode: criteo
 
-biddercode_longer_than_12: false
+biddercode_longer_than_12: true
```

Comment:
Hey @jsfaure just wanted to check on this - it looks like your biddercode is `criteo` which is 6 letters, but you're changing the value here to say it's more than 12 letters.  I don't understand.  Was this an error?  Or was there some other reason for making this change?

Reply:
Hi @rmloveland , thank you for catching this.
So, yes,you're right, biddercode_longer_than_12 should be set to false.
However, I did it on purpose to hide the ""Default Deal ID" Ad Server Key" section as we don't support Deal.
For sure, it's not a good practice.
So, Ideally, it would be great if we can add another paramter to disable the display of the "Deal" section when this is not supported.
What do you think ?

Reply:
Great idea!  I added some logic to the page template in 05651e.

If you add the param  to the header:

`bidder_supports_deals = false`

Then that "Default Deal ID" section will not appear for your bidder.

Reply:
Thank you ! I will update the pull request in consequence.

Reply:
I updated the PR (push force) to be reviewed.
The code relies on the bidder_supports_deals new flag you just introduced.
Thank you.